frontend/src/styles/components/Maps.scss

```diff
@@ -2,13 +2,7 @@
 This file contains the style overrides for ArcGIS Map components
 */
 
-/* FIXME: Add pulsating animation to the LocationPin under the BaseMap Layer
-div.esri-view-root
-  > div.esri-view-surface.esri-view-surface--inset-outline
-  > div
-  > svg
-  > g
-  > circle {
+//This modifies the custum drawn path in the dom.  It is created in the 'LocationPin.js' file
```

Comment:
*custom and *DOM

Otherwise, this looks good. I can't think of a way around the path duplication at the moment, so this seems like a reasonable solution.